api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -541,7 +541,7 @@ function democracyProposalReducer(
     proposalId: proposal.index,
     networkId: network.id,
     type: proposalTypeEnum.PARAMETER_CHANGE,
-    title: `Preliminary Proposal #${proposal.index}`,
+    title: `Proposal #${proposal.index}`,
```

Comment:
Now refendums and democracies are called the same? Well, guess it kinds of makes sense, yes